ci/jenkins/test.sh

```diff
@@ -184,6 +184,31 @@ function clean_for_windows_install_cni {
     done
 }
 
+function printUnixTimestamp () {
+    runtimeOS="$(uname)"
+    if [[ "$runtimeOS" == "Darwin" ]]; then
+        echo $(date -ju -f "%Y-%m-%dT%H:%M:%SZ" "$1" "+%s")
+    else
+        echo $(date -d "$1" '+%s')
+    fi
+}
+
+function run_clean_kind {
+ echo "=== Cleanup Leftover Kind Cluster in Kind Testbed ==="
+    read -a all_kind_clusters <<< $(kind get clusters)
+    for kind_cluster_name in "${all_kind_clusters[@]}"; do
+        creationTimestamp=$(kubectl get nodes --context kind-$kind_cluster_name -o json -l node-role.kubernetes.io/control-plane | \
+        jq -r '.items[0].metadata.creationTimestamp')
+        creation=$(printUnixTimestamp "$creationTimestamp")
+        now=$(date -u '+%s')
+        diff=$((now-creation))
+        if [[ $diff -gt $((135*60)) ]]; then
```

Comment:
Better to use constant here instead of 135x 60

Reply:
Yes, we can do that but I think we don’t need to introduce a constant here because it is not being used anywhere else. It is used only at one place. So, we can skip declaring a constant.

Reply:
@KMAnju-2021  I think the proposal from @rajnkamr is better for improving code readability. We may need to update this value for other kind jobs in the future so we should not hardcode it in the function.